istio-control/istio-discovery/templates/configmap-envoy.yaml

```diff
@@ -113,19 +109,46 @@ data:
           tls_context:
             require_client_certificate: true
             common_tls_context:
+              alpn_protocols:
+              - h2
+
               validation_context:
                 trusted_ca:
                   filename: /etc/certs/root-cert.pem
 
-              alpn_protocols:
-              - h2
-
+  {{- if .Values.global.sds.enabled }}
```

Comment:
so later here and below ```target_uri```/ token ```filename``` will be read from opts["**"] after pilot agent change get in.

Reply:
yes, waiting for that to get in.